src/main/java/org/omnifaces/util/Components.java

```diff
@@ -265,6 +277,24 @@ public static boolean isRendered(UIComponent component) {
         return true;
     }
 
+    /**
+     * Disable the passed UI component.
+     * @param component The component to disable.
+     * @since 4.5
+     */
+    public static void disableInputComponent( UIComponent component ) {
+        setAttribute(component,"disabled",true);
```

Comment:
Can you please follow the same code format as seen in existing source code? The whitespace is off here.